### PR TITLE
Changes to support renew slice

### PR DESCRIPTION
**Pull request by kthare10**
- Changes to support renew slice
- Renew Time should be specified in UTC in the RENEW_TIME_FORMAT = "%Y-%m-%d %H:%M:%S"
- Max duration for renewal is set to 14 days, anything longer is ignored

### CODE REVIEW

**Comment by ibaldin** [comment]
Can we take off 14 day limit for now? Make it a month

File: fabric_cf/orchestrator/core/orchestrator_handler.py [hunk 1]
```diff
@@ -390,4 +393,80 @@ def get_slice_graph(self, *, token: str, slice_id: str) -> dict:
...
+        Renew a slice
+        :param token Fabric Identity Token
+        :param slice_id Slice Id
+        :param new_lease_end_time: New Lease End Time in UTC in '%Y-%m-%d %H:%M:%S' format
+=       :raises Raises an exception in case of failure
+        :return:
+        """
+        failed_to_extend_rid_list = []
+        try:
+            new_end_time = datetime.strptime(new_lease_end_time, Constants.RENEW_TIME_FORMAT)
```
Comment (ibaldin) [inline fabric_cf/orchestrator/core/orchestrator_handler.py:409]: Are there any exceptions from this? This can't always succeed...

**Comment by kthare10** [comment]
> Can we take off 14 day limit for now? Make it a month

Updated to 30 days

File: fabric_cf/orchestrator/core/orchestrator_handler.py (hunk 1 above)
Comment (ibaldin) [reply fabric_cf/orchestrator/core/orchestrator_handler.py:409]: Oh never mind, this is inside a try block and error in the end. How is the exception communicated back to user though?

Comment (kthare10) [reply fabric_cf/orchestrator/core/orchestrator_handler.py:409]: API fails with 500 error and exception string is returned. I'll change it to return Bad Request error

Comment (ibaldin) [reply fabric_cf/orchestrator/core/orchestrator_handler.py:409]: OK

**Review by ibaldin — APPROVED** [review]

**Pull request merged**